Client/WebFilters/FilterListResourceDownloader.swift

```diff
@@ -523,3 +512,42 @@ private extension AdblockService {
     }
   }
 }
+
+// MARK: - FilterListLanguageProvider - A way to share `defaultToggle` logic between multiple structs/classes
+
+private protocol FilterListLanguageProvider {
+  var languages: [String] { get }
+  var componentId: String { get }
+}
+
+extension FilterList: FilterListLanguageProvider {}
+extension AdblockFilterListCatalogEntry: FilterListLanguageProvider {}
+
+private extension FilterListLanguageProvider {
+  @available(iOS 16, *)
```

Comment:
Why is it iOS 16 specific code?

Reply:
`Locale.Region` is the new API in 16, you should use the the old `Locale.current.regionCode` API instead

Reply:
I use both depending on the version so we don't have deprecation warnings